qemu/tests/cfg/qemu_guest_agent.cfg

```diff
@@ -462,10 +469,15 @@
         - gagent_check_blacklist:
             only Linux
             gagent_check_type = blacklist
-            black_list_spec = BLOCK_RPCS
+            black_list_spec = allow-rpcs
+            black_list_spec_replace = block-rpcs
+            black_list_change_cmd = sed -i 's/${black_list_spec}.*/${black_list_spec_replace}=guest-info"/g' /etc/sysconfig/qemu-ga
             Host_RHEL.m7, Host_RHEL.m8, Host_RHEL.m9.u0, Host_RHEL.m9.u1:
-                black_list_spec = BLACKLIST_RPC
```

Comment:
One minor question, why we remove this ?

Reply:
It's not removed, it was moved to another proper position for the corresponding condition.